src/content/en/tools/chrome-devtools/profile/network-performance/resource-loading.markdown

```diff
@@ -385,122 +388,213 @@ The Frames tab shows messages sent or received over a WebSocket connection. This
+* **Timeline** — Sorts by the start time of each network request. This is 
+  the default sort, and is the same as sorting by the **Start Time** option.
+* **Start Time** — Sorts by the start time of each network request (same 
+  as sorting by the **Timeline** option).
 * **Response Time** — Sorts by each request's response time.
 * **End Time** — Sorts by the time when each request completed.
-* **Duration** — Sorts by the total time of each request. Select this filter to determine which resource takes the longest time to load.
-* **Latency** — Sorts by the time between the start of the request and the beginning of the response. Select this filter to determine which resource takes the longest time to first byte.
-
-To filter the Network table to only show certain types of resources, click one of the content types along the bottom of the panel: **Documents**, **Stylesheets**, **Images**, **Scripts**, **XHR**, **Fonts**, **WebSockets**, and **Other**. In the following screenshot only CSS resources are shown. To view all content types, click the **All** filter button.
-
-![Filter type](imgs/filter-type.png)
-
-## Copy and save network information
-
-Copy and save actions are accessible in each resource's context menu.
-The context menu appears with several actions when you <span class="kbd">Right-click</span> or <span class="kbd">Ctrl</span> + <span class="kbd">Click</span> (Mac only) on a selected resource:
-
-![Right-click on network resource](imgs/right-click.png)
-
-The following menu actions apply to the selected resource:
-
-* **Open Link in New Tab** — Opens the resource in a new tab. You can also double-click the resource name in the Network table.
-* **Copy Link Address** — Copies the resource URL to the system clipboard.
-* **Copy Request Headers** — Copies the HTTP request headers to the system clipboard.
-* **Copy Response Headers** — Copies the HTTP response headers to the system clipboard.
-* **Copy as cURL** — Copies the network request as a
-  [cURL](http://curl.haxx.se/) command string to the system clipboard. See [Copying requests as cURL commands](#copy-requests-as-curl-commands).
-* **Replay XHR** — If the associated request is an XMLHTTPRequest, re-sends the original XHR.
+* **Duration** — Sorts by the total time of each request. Select this 
+  filter to determine which resource takes the longest time to load.
+* **Latency** — Sorts by the time between the start of the request and the 
+  beginning of the response. Select this filter to determine which resource 
+  takes the longest time to first byte (TTFB).
+
+## Filter requests 
+
+The **Network** panel provides numerous ways to filter which resources are 
+displayed. Click the **filters** button
+(![filters button](imgs/filters.png){:.inline})
+to hide or display the **Filters** pane.
+
+Use the content type buttons to only display resources of the selected 
+content type. Tip: hold <kbd>cmd</kbd> and then click on one of these
```

Comment:
The tip needs alt styling to call it out. Should probably use a [note](https://web-central.appspot.com/web/resources/jekyll/useful-and-common-tags?hl=en) type thing here. Also mention Cmd or Ctrl for Windows/Linux.